### PR TITLE
Issue #27: Fix boundary bug in PatchFilter

### PR DESCRIPTION
**Pull request by HuGanghui**
Issue #27: Fix boundary bug in PatchFilter

### CODE REVIEW

File: src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java [hunk 1]
```diff
@@ -74,8 +74,14 @@ private boolean isLineMatching(AuditEvent event) {
...
-                result = aLineRangeList.get(0) < event.getLine()
-                        && event.getLine() < aLineRangeList.get(1);
+                final int startLine = aLineRangeList.get(0);
+                final int endLine = aLineRangeList.get(1);
+                if (startLine == endLine) {
+                    result = startLine == event.getLine();
+                }
+                else {
+                    result = startLine < event.getLine() && event.getLine() <= endLine;
+                }
```
Comment (rdiachenko) [inline src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java:84]:
```
int currentLine = event.getLine();
result = currentLine >= startLine && currentLine <= endLine;
```

Comment (HuGanghui) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java]: done

Comment (rdiachenko) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java]: It is not done. The whole `if-else` block had to be replaced with those 2 lines.

Comment (romani) [reply src/main/java/com/github/checkstyle/patchfilter/SuppressionPatchFilterElement.java]: moved to https://github.com/checkstyle/patch-filters/issues/38, should be done asap

**Pull request merged**